sblib.py

```diff
@@ -169,7 +170,7 @@ def statsBryant(observations, models):
     """
     obsNaNs = np.argwhere(np.isnan(observations)).squeeze()
     modNaNs = np.argwhere(np.isnan(models)).squeeze()
-    if isinstance(observations, np.ma.masked_array) or isinstance(models, np.ma.masked_array):
```

Comment:
lets keep the is instance function, i think that's the proper way to do those comparisons, but totally open to input

Reply:
Fixed.  this line should be identical to the deleted one above.